README.md

```diff
@@ -141,7 +141,7 @@ Add to your `.babelrc`'s plugins array.
 ## Benchmarks
```

Comment:
Good idea. I went overboard because I am often picked up on the lack of information on my commit messages.